### PR TITLE
Missing chart in helm install statement

### PR DESCRIPTION
**Pull request by VDuda**
Usage: (v2.1.3)
  helm install [CHART] [flags]

* Include chart in install statement
* Fix set flag to use strings as required

### CODE REVIEW

**Comment by VDuda** [comment]
I signed it!

File: stable/kube2iam/README.md [hunk 1]
```diff
@@ -58,10 +58,10 @@ The following tables lists the configurable parameters of the kube2iam chart and
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
 
 ```console
-$ helm install --name my-release \
-  --set=extraArgs.base-role-arn=arn:aws:iam::0123456789:role/, \
+$ helm install stable/kube2iam --name my-release \
+  --set="extraArgs.base-role-arn=arn:aws:iam::0123456789:role/, \
     extraArgs.default-role=kube2iam-default, \
-    host.iptables=true,host.interface=cbr0
+    host.iptables=true,host.interface=cbr0"
```
Comment (mgoodness) [inline stable/kube2iam/README.md]: Even this doesn't work correctly. Let's just trim it down to:
```console
$ helm install stable/kube2iam --name my-release \
  --set=extraArgs.base-role-arn=arn:aws:iam::0123456789:role/,host.iptables=true,host.interface=cbr0
```

**Review by mgoodness — CHANGES_REQUESTED** [review]

File: stable/kube2iam/README.md (hunk 1 above)
Comment (VDuda) [reply stable/kube2iam/README.md]: It worked on my end - at least when I checked by running an AWS s3 call through a test container. Will apply the change, should def be trimmed down. 👍

**Review by mgoodness — APPROVED** [review]

**Comment by mgoodness** [comment]
Nit/request: would you mind also changing line 68 so the chart name is listed before the arguments? Consistency FTW.

**Comment by mgoodness** [comment]
@k8s-bot ok to test

**Pull request merged**